corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -91,10 +91,11 @@ hqDefine('geospatial/js/models', [
         };
     };
 
-    var Map = function (usesClusters) {
+    var Map = function (usesClusters, usesStreetsLayers) {
```

Comment:
When would `usesStreetsLayers` be false?

Reply:
It would be false for all the other Geospatial pages instantiating the `Map` class. Since we currently only use the streets layers on the "Case Management" page, there's no need to create the streets layers for other pages.